### DISCUSSION TITLE
S905x3

### DISCUSSION THREAD

**Discussion by h3ld3rkid** [ophub/amlogic-s9xxx-armbian#1576] [Amlogic]
alfter boot with luna. i cant install HA. theres some tutorial for noob ppl?

thnkx

**Comment by ophub** [comment]
armbian-software

**Reply by h3ld3rkid** [reply]
sorry my friend. i dont understand...

**Comment by ophub** [comment]
https://github.com/ophub/amlogic-s9xxx-armbian/blob/main/documents/armbian_software.md

**Reply by h3ld3rkid** [reply]
facing this problem
![1](https://github.com/ophub/amlogic-s9xxx-armbian/assets/14248077/b3143808-098f-4fb6-96d5-bffd14dd306c)
![photo_2023-07-26_23-16-11](https://github.com/ophub/amlogic-s9xxx-armbian/assets/14248077/64b87a7a-0100-434a-95a3-0e107bac67c4)
 with this script https://raw.githubusercontent.com/maxcalavera81/Instalacao_armbian_hassio_TX3/main/instalacao_homeassistant.sh

**Comment by ophub** [comment]
101
108

**Reply by h3ld3rkid** [reply]
the secound image the script dont let me choose nothing
in resume:
after armbian install, i take off the USB, boot the box and run armbian-software? and choose 101 the 108?

its this..

sorry, i'm n00b

**Reply by h3ld3rkid** [reply]
i just put it to work... its possible have a supervioned version instead this version? or install manually?

thnkx @ophub

**Comment by h3ld3rkid** [comment]
now i have this, after run armbian-software and choose, 101 and 108
![Captura de ecrã 2023-07-30, às 10 11 27](https://github.com/ophub/amlogic-s9xxx-armbian/assets/14248077/80ad327d-a4f6-4d09-980d-e0d935263d27)

but in ip:8123 doenst.t show nothing
